pkg/bpf/bpf_kmesh_workload.go

```diff
@@ -83,6 +83,10 @@ func (sc *BpfSockConnWorkload) loadKmeshSockConnObjects() (*ebpf.CollectionSpec,
 		return nil, err
 	}
 
+	if kmeshStartStatus == Reload {
```

Comment:
do we need to check this in BpfKmeshWorkload.Load

Reply:
yes，it's required before every pin action